logger/logger.go

```diff
@@ -153,3 +162,14 @@ func SimpleSetup() error {
 	}
 	return err
 }
+
+var procCmdline = "/proc/cmdline"
+
+func debugEnabledOnKernelCmdline() bool {
```

Comment:
can we add a TODO about:

// TODO: consider generalizing this to snapdenv and having it used by other places that consider SNAPD_DEBUG